### PR TITLE
Docker hub

### PR DESCRIPTION
**Pull request by myleftfoot**
Adding Docker hub configuration setting

### CODE REVIEW

File: src/server/mappers/swarm-client.js [hunk 1]
```diff
@@ -69,10 +69,11 @@ async function removeLabel({ swarmhost, nodeId, label }) {
  * @param  {[type]} options.accessKeyId     [description]
  * @param  {[type]} options.secretAccessKey [description]
  * @param  {[type]} options.stack           [description]
+ * @param  {[type]} options.dockerHub       [description]
  * @return {[type]}                         [description]
  */
-async function deployStack({ swarmhost, accessKeyId, secretAccessKey, stack }) {
-  return await put(`http://${swarmhost}:4111/api/stacks/${stack}`, { json: { accessKeyId, secretAccessKey }});
+async function deployStack({ swarmhost, accessKeyId, secretAccessKey, stack, dockerHub }) {
+  return await put(`http://${swarmhost}:4111/api/stacks/${stack}`, { json: { accessKeyId, secretAccessKey, dockerHub }});
```
Comment (myleftfoot) [inline src/server/mappers/swarm-client.js]: not from my code but I think this should be something like

${swarmhost}/api where swarmhost is http://host:port

**Comment by adampaulberg** [comment]
I don't know much about swarm, but nothing here seems to be breaking changes from what i can tell.

**Comment by jakebrown58** [comment]
did you intend to get the swarm stuff pulled into master with this request?

**Comment by myleftfoot** [comment]
@jakebrown58 my intent is to merge it into the swarm branch. At some point @bmancini55 and I will want to merge it into master

**Comment by jakebrown58** [comment]
cool - i missed that at first.  i'll leave this merge up to Brian - i'm not up to speed on the dockerhub / swarm branch.

**Pull request merged**